src/ZendeskApi.Client/Resources/UsersResource.cs

```diff
@@ -259,6 +259,27 @@ public async Task<UserResponse> UpdateAsync(UserUpdateRequest user)
             }
         }
 
+        public async Task<UserResponse> CreateOrUpdateAsync(UserCreateRequest user)
+        {
+            using (_loggerScope(_logger, "CreateOrUpdateAsync"))
+            using (var client = _apiClient.CreateClient(ResourceUri))
+            {
+                var response = await client.PostAsJsonAsync("create_or_update", new UserRequest<UserCreateRequest>(user)).ConfigureAwait(false);
+
+                if (response.StatusCode != HttpStatusCode.Created && response.StatusCode != HttpStatusCode.OK)
```

Comment:
Should this not be OR instead of AND?

Reply:
That would be unfortunate:
201 != 201 || 201 != 200 => true, throw
200 != 201 || 200 != 200 => true, throw
400 != 201 || 400 != 200 => true, throw
compared to:
201 != 201 && 201 != 200 => false
200 != 201 && 200 != 200 => false
400 != 201 && 400 != 200 => true, throw

Reply:
Makes sense, thanks for clarifying!